components/mpas-ocean/cime_config/buildnml

```diff
@@ -568,21 +568,13 @@ def buildnml(case, caseroot, compname):
             lines.append('')
             lines.append('<immutable_stream name="mesh"')
             lines.append('                  type="none"')
-            if ocn_grid.startswith("oRRS1") or ocn_grid.startswith("FRIS") \
-                    or ocn_grid.startswith("RRSwISC6"):
```

Comment:
Anyone know why these exceptions were carved out?

Reply:
I can restore them if they are still required for certain cases.

Reply:
pnetcdf with cdf5 is now our default.  When ADIOS is chosen but an input file is only .nc, the pnetcdf and cdf5 will be used.  So I don't think this exception is needed.

Reply:
It seems that io_type="pnetcdf,cdf5" (MPAS_IO_PNETCDF5, uses PIO_64BIT_DATA) and io_type="pnetcdf" (MPAS_IO_PNETCDF, uses PIO_64BIT_OFFSET) are different in MPAS framework. And PIO_NETCDF_FORMAT in CIME settings (64bit_data by default) is never used currently.

Reply:
Also, for some low-res meshes (e.g. ne30pg2_r05_IcoswISC30E3r5), they might also need 64bit_data for certain large output files, such as mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc.

Reply:
PIO_NETCDF_FORMAT is now taken into account to set io_type to either "pnetcdf,cdf5" or "pnetcdf" as appropriate.